### PR TITLE
[Doc] Update add_datasets docs

### PR DESCRIPTION
**Pull request by csatsurnh**
as title

### CODE REVIEW

File: docs/zh_cn/advanced_guides/add_datasets.md
Comment (MeowZheng) [inline docs/zh_cn/advanced_guides/add_datasets.md:1]: there is no `split` of dataset.

Comment (MeowZheng) [inline docs/zh_cn/advanced_guides/add_datasets.md:1]: we might add more explaination about why there is no test sub-dataset
https://github.com/open-mmlab/mmsegmentation/issues/2752

Comment (MeowZheng) [inline docs/zh_cn/advanced_guides/add_datasets.md:1]: there is no  args `classes` and `palette` of dataset

Comment (MeowZheng) [inline docs/zh_cn/advanced_guides/add_datasets.md:1]: I think we should tell users how to build their own datasets, like implementing a dataset class

File: docs/en/advanced_guides/add_datasets.md [hunk 1]
```diff
@@ -26,26 +26,14 @@ An example of file structure is as followed.
 
 A training pair will consist of the files with same suffix in img_dir/ann_dir.
 
-If `split` argument is given, only part of the files in img_dir/ann_dir will be loaded.
-We may specify the prefix of files we would like to be included in the split txt.
+Some datasets don't release test sub-dataset or don't release the ground truth of test sub-dataset, and we cannot evaluate model locally without ground truth of test sub-dataset, so we ignore test sub-dataset in config files.
```
Comment (xiexinch) [inline docs/en/advanced_guides/add_datasets.md:29]:
```suggestion
Some datasets don't release the test set or don't release the ground truth of the test set, and we cannot evaluate models locally without the ground truth of the test set, so we set the validation set as the default test set in config files.
```

File: docs/zh_cn/advanced_guides/add_datasets.md [hunk 2]
```diff
@@ -58,79 +47,29 @@ MMSegmentation 同样支持混合数据集去训练。
...
-           dataset_A_train,
-           dataset_B_train
-       ],
-       val = dataset_A_val,
-       test = dataset_A_test
-       )
-   ```
-
-一个更复杂的例子如下：分别重复 `Dataset_A` 和 `Dataset_B` N 次和 M 次，然后再去拼接重复后的数据集
+如果要连接的数据集不同，可以按如下方式连接数据集配置。
```
Comment (xiexinch) [inline docs/zh_cn/advanced_guides/add_datasets.md:62]:
```suggestion
如果要拼接不同的数据集，可以按如下方式连接数据集配置。
```

**Review by xiexinch — APPROVED** [review]

**Pull request merged**